python/mobilecoin/cli.py

```diff
@@ -246,11 +246,12 @@ def status(self):
         print()
         print('Transaction Fees:')
         for token in TOKENS:
-            amount = Amount.from_storage_units(
-                network_status['fees'][str(token.token_id)],
-                token
-            )
-            print(indent(amount.format(), '  '))
+            if str(token.token_id) in network_status['fees']:
+                amount = Amount.from_storage_units(
+                    network_status['fees'][str(token.token_id)],
+                    token
+                )
+                print(indent(amount.format(), '  '))
```

Comment:
The change is good enough as is

python does not do *any* optimizations to code logic
this means it will:
- `str(token.token_id)` twice
- likely look for the string in `network_status['fees']` twice

I doubt this code path is a performance bottleneck to need the change, just an FYI if one isn't aware
```suggestion
            token_fee = network_status['fees'].get(str(token.token_id))
            if token_fee is not None:
                amount = Amount.from_storage_units(
                    token_fee,
                    token
                )
                print(indent(amount.format(), '  '))
```